examples/js/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@algolia/autocomplete-js": "^1.0.0-alpha.32",
+    "@algolia/autocomplete-plugin-recent-searches": "^1.0.0-alpha.28",
```

Comment:
We need to bump our internal deps to `.32`.

Reply:
https://github.com/algolia/autocomplete.js/pull/326/commits/3e5b77b1b72653926c3490497ae53848b1f09236